frontend/src/slices/users/actions.ts

```diff
@@ -0,0 +1,26 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+
+import { AppRoute } from '#libs/enums/enums.js';
+import { StorageKey } from '#libs/packages/storage/storage.js';
+import { type AsyncThunkConfig } from '#libs/types/types.js';
+import { type UserDeleteRequestDto } from '#packages/users/users.js';
+import { actions as appActions } from '#slices/app/app.js';
+
+import { name as sliceName } from './users.slice.js';
+
+const deleteUser = createAsyncThunk<
+  unknown,
```

Comment:
```suggestion
  number,
```
You return id. If you don't need it, just make it void